doc/source/accuracy_precision.md

```diff
@@ -0,0 +1,115 @@
+(accuracy-precision)=
+
+# Grasping accuracy and precision
+
+Below, a small guide explaining what accuracy and precision are, and their relation to elevation data (or any spatial data!).
+
+## Why do we need to understand accuracy and precision?
+
+Elevation data comes from a wide range of instruments (optical, radar, lidar) acquiring in different conditions (ground,
+airborne, spaceborne) and relying on specific post-processing techniques (e.g., photogrammetry, interferometry).
+
+While some complexities are specific to certain instruments and methods, all elevation data generally possesses:
+
+- a [ground sampling distance](https://en.wikipedia.org/wiki/Ground_sample_distance), or pixel size, **that does not necessarily represent the underlying spatial resolution of the observations**,
+- a [georeferencing](https://en.wikipedia.org/wiki/Georeferencing) **that can be subject to shifts, tilts or other deformations** due to inherent instrument errors, noise, or associated processing schemes,
+- a large number of [outliers](https://en.wikipedia.org/wiki/Outlier) **that remain difficult to filter** as they can originate from various sources (e.g., blunders, clouds).
+
+All of these factors lead to difficulties in assessing the reliability of elevation data, required to
+perform additional quantitative analysis, which calls for defining the concepts relating to accuracy and precision for elevation data.
+
+## Accuracy and precision of elevation data
+
+### What are accuracy and precision?
+
+[Accuracy and precision](https://en.wikipedia.org/wiki/Accuracy_and_precision) describe random and systematic errors, respectively.
```

Comment:
Flip "random and systematic", as accuracy is systematic and precision is random.

Reply:
Good catch! Fixed